packages/api/src/controllers/taskController.js

```diff
@@ -59,6 +59,19 @@ const taskController = {
         user_id,
       );
 
+      if (newAssigneeUserId === null) {
+        const farmManagementObjs = await userFarmModel.getFarmManagementByFarmId(farm_id);
+        const farmManagement = farmManagementObjs.map((obj) => obj.user_id);
+        await sendTaskNotification(
+          farmManagement,
+          oldAssigneeUserId,
```

Comment:
I believe this should be `user_id` because currently, the notification message says "oldAssigneeUser" has marked <task> as unassigned. But the editor here is currently authenticated user not the old assignee user.

Reply:
Good catch - I totally missed that! This should be fixed with my new commit.